src/tests/parse.test.ts

```diff
@@ -0,0 +1,22 @@
+import assert from "assert/strict";
+import { BigNumber } from "bignumber.js";
+import parseAmount from "../helpers/parser.js";
+
+function testNormalBigAmount() {
+  const rawFungibleTokenAmount = "0x00000000000000000de0b6b3a7640000";
```

Comment:
Just for reference this value was extracted from Turing Indexer under [this tx](https://avail-turing.subscan.io/extrinsic/0xbed5e37a6cc93b52a3133f18243b95f4c8fedf4475685258c6a39b5ebcfb8ca3)

**GQL query:**
```gql
query {
  extrinsics(
    filter: {
      call: { equalTo: "sendMessage" }
      success: { equalTo: true }
      blockHeight: { equalTo: "564024" }
    }
    first: 2
    orderBy: BLOCK_HEIGHT_ASC
  ) {
    nodes {
      id
      txHash
      module
      call
      blockHeight
      success
      isSigned
      extrinsicIndex
      hash
      timestamp
      signer
      signature
      fees
      nonce
      argsName
      argsValue
      nbEvents
    }
  }
}
```